improver_tests/calibration/reliability_calibration/test_AggregateReliabilityCalibrationTables.py

```diff
@@ -127,29 +126,62 @@ def test_aggregating_multiple_cubes(self):
         """Test of aggregating two cubes without any additional coordinate
         collapsing."""
 
-        plugin = Plugin()
-        result = plugin.process([self.reliability_cube, self.reliability_cube])
+        frt = 'forecast_reference_time'
+        expected_points = self.different_frt.coord(frt).points
+        expected_bounds = [[self.reliability_cube.coord(frt).bounds[0][0],
+                            self.different_frt.coord(frt).bounds[-1][1]]]
 
+        plugin = Plugin()
+        result = plugin.process([self.reliability_cube, self.different_frt])
+        print(result)
```

Comment:
I think this `print` statement isn't needed.

Reply:
Everyone's against my random print statements.